extender/types.go

```diff
@@ -17,6 +19,18 @@ type Server struct {
 	Scheduler
 }
 
+type NotBindingExtender struct{}
```

Comment:
These types and functions need comments for export. Our github action isn't picking this up - golint needs to be configured with lower sensitivity - but I'm seeing it when I run golint locally.

Reply:
removed